datasets/bdi_benchmark/BDI-With-Matching.recipe.yml

```diff
@@ -0,0 +1,42 @@
+# ========================================================================================
+# BDI Performance Test - Test Data with 100% Matching to existing Account and Contact
+#
+# run it like this:
+#
+# cci task run generate_and_load_from_yaml -o num_records_tablename DataImport__c -o num_records 10000 -o generator_yaml datasets/bdi_benchmark/BDI-With-Matching.recipe.yml --org whatever
+
+- object: General_Accounting_Unit__c
+  nickname: the_gau
+  just_once: True
+  fields:
+      Name: General Fund
+
+- object:  DataImport__c      # Match Existing Contact and Account
+  fields:
+      Contact1_Lastname__c: BDITEST ${{batchnum}} ${{id}}
+      Account1_Country__c: Tuvalu
+      Account1_Street__c: Cordova Street
+      Donation_Donor__c: Contact1
+      Donation_Amount__c: ${{id * 100}}
+      Donation_Record_Type_Name__c: Donation
+      Donation_Date__c:
+        date_between:
+          start_date: -0d
+          end_date: +90d
+      Opportunity_Contact_Role_1_Role__c: Influencer
+      GAU_Allocation_1_GAU__c:
+        reference:
+          - the_gau
+      GAU_Allocation_1_Percent__c: 100
+      Payment_Method__c: Check
+  friends:
+      - object: Contact  # Contact for matching
+        fields:
+            FirstName:
+              fake: first_name
+            Email:
+              fake: free_email
+            Phone:
+              fake: phone_number
+            LastName: ${{DataImport__c.Contact1_Lastname__c}}
```

Comment:
Double-checking: last-name alone is sufficient to generate the match?

Reply:
I seem to remember that worked, but to be honest it's been a little while. I'll spin up a new scratch org to re-test and confirm.